server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -642,5 +642,14 @@ export function configureWebSocketServices(
 			}
 			disposers.splice(0, disposers.length);
 		});
+
+		socket.on(
+			"disconnect_document",
+			(clientId: string, documentId: string, errorType?: string) => {
+				Lumberjack.error(
```

Comment:
should this skip logging if errorType is undefined. it seems like logging every disconnect could produce a significant amount of data